.github/workflows/callable.release.yml

```
@@ -0,0 +1,50 @@
+
+      - uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'adopt'
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+
+      - name: Publish to maven repository
+        run: ./gradlew publishPluginPublicationToHangar publish
+        env: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)